packages/@ourworldindata/grapher/src/stackedCharts/StackedAreaChart.tsx

```diff
@@ -126,7 +130,7 @@ class Areas extends React.Component<AreasProps> {
 
     @computed private get areas(): JSX.Element[] {
         const { placedSeriesArr } = this
-        const { dualAxis } = this.props
+        const { dualAxis, hoveredAreaName } = this.props
```

Comment:
NB: this on-hover highlighting behavior didn't exist previously. Non-hovered area slices should fade out and the highlighted series row should be emboldened in the tooltip